### PR TITLE
Made new script to update current intechopen links

### PR DESCRIPTION
**Pull request by mitri-slory**

### CODE REVIEW

File: scripts/updateOldInTechOpenLinks.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    doabProcess = DOABProcess('single', None, None, None)
+
+    identRegex = r'intechopen.com\/storage\/books\/([\d]+)'
+
+    for record in dbManager.session.query(Record) \
+        .filter(Record.source != 'doab') \
+        .filter(Record.publisher == '{IntechOpen||,IntechOpen||}').all():
+            
+            match = re.search(identRegex, record.has_part)
```
Comment (mwbenowitz) [inline scripts/updateOldInTechOpenLinks.py:37]: `record.has_part` comes back as an array I think, so you might need to iterate through this and do the check on each value. The regex test looks good though!

**Review by mwbenowitz — COMMENTED** [review]
Looks great, just want to make sure we're handling the `record.has_part` field correctly. If we are then happy to approve this!

**Review by mwbenowitz — APPROVED** [review]
Looks great!

File: scripts/updateOldInTechOpenLinks.py [hunk 2]
```diff
@@ -42,8 +41,10 @@ def main():
 
             if match == None:
                 doabProcess.importSingleOAIRecord(record.source_id)
-                
-            
+                doabProcess.saveRecords()
+                #Make session record a set after updating record in database to reduce memory space
+                record = set()
```
Comment (mwbenowitz) [inline scripts/updateOldInTechOpenLinks.py:46]: should be `doabProcess.records = set()` here! That will make sure that the process class doesn't build up a huge object containing all the records we've processed.

Comment (mitri-slory) [reply scripts/updateOldInTechOpenLinks.py:46]: Appreciate the correction!

**Review by mwbenowitz — APPROVED** [review]
👍

**Pull request merged**